diff --git a/support/recompile_check.py b/support/recompile_check.py
index c991198e..999896b6 100644
--- a/support/recompile_check.py
+++ b/support/recompile_check.py
@@ -66,6 +66,10 @@ recompile_list = [
     gentest('test_multiple_generator_input_arguments.clsp')
 ]
 
+here = Path(__file__).parent.resolve()
+root = here.parent
+hashes_path = root.joinpath("chia/wallet/puzzles/deployed_puzzle_hashes.json")
+
 for recompile_entry in recompile_list:
     if 'dirname' in recompile_entry and 'fname' in recompile_entry:
         dirname = recompile_entry['dirname']
