--- a/setuptools/wheel.py	2022-05-10 08:37:19.000000000 -0700
+++ b/setuptools/wheel.py	2022-05-13 09:08:07.805319336 -0700
@@ -92,7 +92,7 @@
 
     def install_as_egg(self, destination_eggdir):
         '''Install wheel as an egg directory.'''
-        with zipfile.ZipFile(self.filename) as zf:
+        with zipfile.ZipFile(self.filename, strict_timestamps=False) as zf:
             self._install_as_egg(destination_eggdir, zf)
 
     def _install_as_egg(self, destination_eggdir, zf):
diff --git a/setuptools/command/bdist_egg.py b/setuptools/command/bdist_egg.py
index 11a1c6b..f7593a6 100644
--- a/setuptools/command/bdist_egg.py
+++ b/setuptools/command/bdist_egg.py
@@ -447,7 +447,7 @@ def make_zipfile(zip_filename, base_dir, verbose=0, dry_run=0, compress=True,
 
     compression = zipfile.ZIP_DEFLATED if compress else zipfile.ZIP_STORED
     if not dry_run:
-        z = zipfile.ZipFile(zip_filename, mode, compression=compression)
+        z = zipfile.ZipFile(zip_filename, mode, compression=compression, strict_timestamps=False)
         for dirname, dirs, files in sorted_walk(base_dir):
             visit(z, dirname, files)
         z.close()
